contracts/lib/arbitration/IArbitrable.sol

```diff
@@ -15,21 +15,20 @@ import "../../lib/standards/ERC165.sol";
 *      since we are using different solidity versions.
```

Comment:
I've changed this here to align it with the other ERC165 implementations we have, but have some doubts.

I am not sure if this and `IArbitrator` are actually useful to include here, as Disputables don't need to have any concept of these interfaces. Perhaps they should be moved to the Agreement implementation instead?

Reply:
It's being used in the arbitrable events

Reply:
Update: after an offline discussion about this, we decided to trim the `IArbitrable` interface removing all the arbitration-related info, please review commit https://github.com/aragon/aragonOS/pull/612/commits/bbd03046a11bf605b68a33acc6067500e550ca81